.github/release-drafter.yml

```
@@ -0,0 +1,9 @@
+_extends: jellyfin-meta-plugins
+
+template: |
+  <!-- Optional: add a release summary here -->
+  [Plugin build can be downloaded here](https://repo.jellyfin.org/releases/plugin/jellyfin-plugin-fanart/jellyfin-plugin-fanart_$NEXT_MAJOR_VERSION.0.0.0.zip).
```

jellyfin-plugin-* are the old path, they go in the dir without now (slugified plugin name)
```suggestion
  [Plugin build can be downloaded here](https://repo.jellyfin.org/releases/plugin/fanart/fanart_$NEXT_MAJOR_VERSION.0.0.0.zip).
```
https://repo.jellyfin.org/releases/plugin/fanart/fanart_6.0.0.0.zip
https://github.com/jellyfin/jellyfin-plugin-fanart/blob/778125d1b4f3ed4e88edaead9e71b0f59bcbb7a1/build.yaml#L2